library/VTKExtensions/Rendering/vtkF3DRendererWithColoring.cxx

```diff
@@ -78,7 +78,8 @@ void vtkF3DRendererWithColoring::SetImporter(vtkF3DGenericImporter* importer)
 //----------------------------------------------------------------------------
 void vtkF3DRendererWithColoring::Initialize(const std::string& up)
 {
-  this->Superclass::Initialize(up);
+  this->RemoveAllViewProps();
```

Comment:
@mwestphal in case you wonder why I moved this. I need to add the scalar bar before the cheatsheet, otherwise the scalar bar would be above the cheatsheet which look bad.

Reply:
alright, the new logic looks fine.